.pre-commit-config.yaml

```diff
@@ -20,10 +21,15 @@ repos:
         exclude: >
           (?x)^(
             deploy/twingate-operator/templates/.*|
+            deploy/test/golden/.*|
           )$
       - id: debug-statements
       - id: destroyed-symlinks
       - id: end-of-file-fixer
+        exclude: >
+          (?x)^(
+            deploy/test/golden/.*|
```

Comment:
We dont want fixes or golden test files - if pre-commit changes them, the test will fail